charts/nexus-iq/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: nexus-iq-server
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 161.0.0
+version: 161.0.1
```

Comment:
This does not require any update (unless you want to publish the chart manually to artifacthub immediately). The next planned release (162) will update this line and publish the chart automatically to artifacthub.

Reply:
As 162 is already released, have to perform the changes to new version now. 
Would need a new chart version, like 162.0.1